packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -776,31 +797,31 @@ export class MgtPersonCard extends MgtTemplatedComponent {
       );
     }
 
-    // To be included when support for direct messaging is added
-    // ${this.internalPersonDetails.id !== this._me.id && MgtPersonCard.config.isSendMessageVisible
-    //   ? html`
-    //       <div class="quick-message">
-    //         <input
-    //           type="text"
-    //           class="quick-message__input"
-    //           placeholder="Message ${this.internalPersonDetails.displayName}"
-    //           .value=${this._chatInput}
-    //           @input=${(e: Event) => {
-    //             this._chatInput = (e.target as HTMLInputElement).value;
-    //           }}
-    //         />
-    //         <button class="quick-message__send" @click=${() => this.sendQuickMessage()}>
-    //           ${getSvg(SvgIcon.Send)}
-    //         </button>
-    //       </div>
-    //     `
-    //   : null}
-
     return html`
```

Comment:
Does this section render anything except the `{compactTemplates}`?
I'm not a fan of having large chunks of commented out code, I find them distracting, and it becomes hard to tell if it's something that needs to be removed or something you forgot to undo in the process of testing.

Reply:
It had been implelemnted previously in next/fluentui branch but we no longer are adding this feature. Will remove.